_includes/footer.html

```diff
@@ -0,0 +1,34 @@
+<footer>
+
+    <div class="ui vertical footer inverted segment">
+        <div class="ui container">
+            <div class="ui divided very relaxed inverted three columns grid">
+                <div class="column">
+                    <h2 class="footer-heading">{{ site.title }}</h2>
+
+                    <div class="footer-col footer-col-1">
+                        <ul class="contact-list">
+                            <li>{{ site.title }}</li>
```

Comment:
`site.title` repeated twice in quick succession - makes footer redundant and uninformative; maybe replace text under contact-list by e.g. "Incubateur de services numériques"